packages/node/src/lib/log.ts

```diff
@@ -72,6 +81,56 @@ function patchResponse(res) {
   };
 }
 
+/**
+ * For `text/*` requests Express doesn't give us a native way to retrieve data out of the payload
+ * without using the `body-parser` middleware so we need to workaround it and access that obtain
+ * that data ourselves.
+ *
+ * For `application/vnd.api+json` types of requests, Express doesn't recognize them as being JSON,
+ * resulting in `req.body` being empty. Frustratingly enough `req.is('json')` also doesn't work so
+ * we need to do our own check to look if it's got `+json` and then surface that potential JSON
+ * payload accordingly.
+ *
+ * And if you can believe it or not, Express also doesn't process `x-www-form-urlencoded` payloads
+ * into `req.body` for us without the `body-parser` middleware.
+ *
+ * @see {@link https://stackoverflow.com/a/12497793}
+ * @see {@link https://stackoverflow.com/a/58568473}
+ * @param {IncomingMessage} req
+ */
+function patchRequest(req: ExtendedIncomingMessage) {
+  // If we already have a body then whatever framework we're being run inside of is able to
+  // handle these requests and we can rely on `req.body` instead hacky workarounds.
+  if (req.body !== undefined) {
+    return;
+  }
+
+  if (isRequest(req, 'text/*')) {
```

Comment:
This `isRequest` library I made is a copy of the Express `req.is` code. Uses the same internal `type-is` library to do content type determination.